imports/plugins/core/accounts/client/containers/addressBookContainer.js

```diff
@@ -0,0 +1,212 @@
+    Meteor.call("accounts/addressBookRemove", _id, (error, result) => {
+      if (error || !result) {
+        reject(i18next.t("addressBookGrid.cantRemoveThisAddress", { err: error.message }));
+      } else {
+        resolve(result);
+      }
+    });
+  });
+}
+
+/**
+ * @method addAddress
+ * @summary helper function that adds an address in the account's addressBook via a meteor method.
+ * @since 2.0.0
+ * @param {Object} address - address to be added.
+ * @return {Promise}
+ */
+function addAddress(address) {
+  return new Promise((resolve, reject) => {
+    Meteor.call("accounts/validateAddress", address, (error, result) => {
+      if (error || !result || !result.validated) reject(i18next.t("addressBookAdd.failedToAddAddress", { err: error.message }));
+      Meteor.call("accounts/addressBookAdd", address, (err, res) => {
+        if (err || !res) {
+          reject(i18next.t("addressBookAdd.failedToAddAddress", { err: err.message }));
+        } else {
+          resolve(res);
+        }
+      });
+    });
+  });
+}
+
+/**
+ * @method onError
+ * @summary helper function that shows an error message in an alert toast.
+ * @since 2.0.0
+ * @param {Object} errorMessage - error message object.
+ */
+function onError(errorMessage) {
+  Alerts.toast(errorMessage, "error");
+}
+
+const wrapComponent = (Comp) => (
+  class AddressBookContainer extends Component {
+    static propTypes = {
+      /**
+       * array of address objects
+       */
+      addressBook: PropTypes.arrayOf(PropTypes.shape({
+        _id: PropTypes.String,
+        fullName: PropTypes.String,
+        address1: PropTypes.String,
+        addresss2: PropTypes.String,
+        postal: PropTypes.String,
+        city: PropTypes.String,
+        region: PropTypes.String,
+        country: PropTypes.String,
+        phone: PropTypes.String,
+        isBillingDefault: PropTypes.Bool,
+        isShippingDefault: PropTypes.Bool,
+        isCommercal: PropTypes.Bool
+      })),
+      /**
+       * country options for select
+       */
+      countries: PropTypes.arrayOf(PropTypes.shape({
+        label: PropTypes.String,
+        value: PropTypes.String
+      })),
+      /**
+       *  Heading content for address book
+       */
+      heading: PropTypes.shape({
+        /**
+         * Heading title
+         */
+        defaultValue: PropTypes.String,
+        /**
+         * i18nKey for heading title
+         */
+        i18nKey: PropTypes.String,
+        /**
+         * If in checkout view, addressbook checkout step position and icon className
+         */
+        checkout: PropTypes.shape({
+          icon: PropTypes.String,
+          position: PropTypes.Number
+        })
+      }),
+      /**
+       * regions by county
+       */
+      regionsByCountry: PropTypes.shape({
+        countryCode: PropTypes.arrayOf(PropTypes.shape({
+          label: PropTypes.String,
+          value: PropTypes.String
+        }))
+      })
+    }
```

Comment:
This `regionsByCountry` prop type should be fixed, too. However, because this is a very lightweight wrapper, I'd suggest not setting the explicit propTypes here but rather grabbing them off Comp.

```js
static propTypes = Comp.propTypes
```